plugin/trino-faker/src/main/java/io/trino/plugin/faker/FakerConnector.java

```diff
@@ -126,6 +126,17 @@ public List<PropertyMetadata<?>> getSchemaProperties()
                         null,
                         defaultLimit -> checkProperty(1 <= defaultLimit, INVALID_SCHEMA_PROPERTY, "default_limit value must be equal or greater than 1"),
                         false),
+                doubleProperty(
+                        SchemaInfo.SEQUENCE_MIN_DISTINCT_VALUES_RATIO,
+                        """
+                                Minimum ratio of distinct values of a column to total number of rows in a table to treat the columns as a sequence
+                                when creating a table in this schema using existing data. Set to a value greater than 1 to disable using sequences""",
```

Comment:
I am not sure if this is proper multiline string formatting ;P

Reply:
It was not, good catch lol